srcpkgs/webkit2gtk/template

```diff
@@ -132,21 +134,132 @@ pre_configure() {
 	esac
 }
 
+post_configure() {
+	mkdir -p webkit2gtk-41
+	pushd webkit2gtk-41
+		configure_args=${configure_args//-DUSE_SOUP2=ON/}
+		configure_args=${configure_args//-DENABLE_WEBDRIVER=OFF/}
+		do_configure
+	popd
+	mkdir -p webkit2gtk-50
+	pushd webkit2gtk-50
+		configure_args+=" -DUSE_GTK4=ON -DENABLE_WEBDRIVER=OFF"
+		do_configure
+	popd
+}
+
+pre_build() {
+	export NINJA_STATUS="[1/3][%f/%t] "
+}
```

Comment:
Can this be moved to a top-level export to avoid another function? Or just leave it off; while the 2/3 and 3/3 prefixes are useful later, lack of a prefix can just imply 1/3.